docs/components/best-practices/architecture/sizing-your-environment.md

```diff
@@ -233,4 +235,4 @@ If you are in doubt about which package to choose, you can do a load test with a
 
 This is recommended if you exceed the above numbers of three million process instances per day.
 
-You can look at the [Zeebe benchmark project](https://github.com/camunda-cloud/zeebe/blob/develop/benchmarks/setup/README.md#benchmarking-camunda-cloud-saas). While this project will not run out-of-the-box (e.g. you need need to build starter and worker code yourself and use self-created docker images), you can use it as a starting point for own endavours.
+You can look at the [Camunda 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark). While this project will not run out-of-the-box (e.g. you need need to build starter and worker code yourself and use self-created docker images), you can use it as a starting point for own endavours.
```

Comment:
To clean up a few typos, would rewrite as:

**Take a look at the [Camunda Platform 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark). While this project will not run out of the box (you need need to build starter and worker code yourself and use self-created Docker images), you can use it as a starting point for your own projects.**

Thoughts?

Reply:
Works - just realized it is wrong / outdated :-) Lets's simplify to the following - WDYT? 

Take a look at the [Camunda Platform 8 benchmark project](https://github.com/camunda-community-hub/camunda-8-benchmark) as a starting point for your own benchmarks.